src/app/hero.service.ts

```diff
@@ -0,0 +1,106 @@
+import { Injectable } from '@angular/core';
```

Comment:
It'd be nice to see the service files a bit more organised, at the moment the root of the `/src` directory is looking like a bit of a dumping ground.

It'd be cleaner if only the app components were left in the root, then the services can be moved to another folder, maybe a structure like `/services/hero/hero.service.ts` would be good?